trailblazer/server/api.py

```diff
@@ -131,6 +134,17 @@ def update_analysis(
         return jsonify(error=str(error)), HTTPStatus.BAD_REQUEST
 
 
+@blueprint.route("/summary", methods=["GET"])
+@inject
+def get_summaries(analysis_service: AnalysisService = Provide[Container.analysis_service]):
+    try:
+        request_data: SummariesRequest = parse_summaries_request(request)
+        response: SummariesResponse = analysis_service.get_summaries(request_data)
+        return jsonify(response.model_dump()), HTTPStatus.OK
+    except ValidationError as error:
+        return jsonify(error=str(error)), HTTPStatus.BAD_REQUEST
```

Comment:
Would we want to add some validation that we actually have some analyses/order_ids in place?

Reply:
I think it is fine to get an empty list as a response. The caller does not request a specific resource and should receive an error if there are none

Reply:
If the caller does not provide any order id:s I think it is fine that we return an empty successful response.
I also don't think it is an error that there are no analyses available for the provided orders